nominate/templates/nominate/nominate.html

```diff
@@ -4,43 +4,56 @@
 {% load i18n %}
 {% block content %}
     <div class="panel scrollable-div">
+          <div class="sticky-top d-inline-flex justify-content-end vote-email float-end">
+            <form action="{% url 'election:email-my-nominations' election_id=election.slug %}" method="post">
+              {% csrf_token %}
+              <button type="submit" class="btn btn-primary">{% translate "Email my nominations to me" %}</button>
+            </form>
+          </div>
         <div class="d-flex justify-content-center h-100 pt-3 pt-md-0">
-            <div id="login" class=col-md-9 col-lg-7 col-xl-6">
+          <!-- NOTE: should the id be login here? If so why? 
+            Also check widths 9,7,6 ....
+          -->
+            <div id="login" class="col-md-9 col-lg-7 col-xl-6">
                 <div class="row">
-                    <div class="col-12 col-lg-9">{% include "nominate/bits/nomination_instructions.html" %}</div>
-                    <div class="col-12 col-lg-3">
+                    <div class="d-flex-row">{% include "nominate/bits/nomination_instructions.html" %}</div>
+                    <!-- <div class="col-12 col-lg-3">
                         <form action="{% url 'election:email-my-nominations' election_id=election.slug %}"
                               method="post">
                             {% csrf_token %}
                             <button type="submit" class="btn btn-primary">{% translate "Email my nominations to me" %}</button>
                         </form>
-                    </div>
+                    </div> -->
```

Comment:
No sense in leaving this commented code in, unless you have a specific reason.

Reply:
Yep, makes sense to me